lib/charms/mongodb/v0/shards_interface.py

```diff
@@ -126,6 +130,12 @@ def pass_hook_checks(self, event: EventBase) -> bool:
             event.defer()
             return False
 
+        # adding/removing shards while a backup/restore is in progress can be disastrous
+        pbm_status = self.charm.backups.get_pbm_status()
+        if isinstance(pbm_status, MaintenanceStatus):
+            event.defer("Cannot add/remove shards while a backup/restore is in progress.")
```

Comment:
I think here should be 
`logger.info(message)` in addition to `event.defer(message)` because `event.defer` does not accept any params 😄